### PR TITLE
New JohnCena Goodie

### PR DESCRIPTION
**Pull request by JoshuaRLi**
https://duck.co/ia/view/john_cena

### CODE REVIEW

**Comment by daxtheduck** [comment]
@JoshuaRLi Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: t/JohnCena.t [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+zci answer_type => "john_cena";
+zci is_cached   => 1;
+
+ddg_goodie_test(
+    ['DDG::Goodie::JohnCena'],
+
+    'who is champ' => test_zci(
+        'JOHN CENA',
+        structured_answer => {
+            input     => [],
```
Comment (moollaza) [inline t/JohnCena.t]: @JoshuaRLi looks like this needs to be updated to match the new `structured_answer` above. After that this is good to merge!

File: lib/DDG/Goodie/JohnCena.pm [hunk 2]
```diff
@@ -0,0 +1,42 @@
+package DDG::Goodie::JohnCena;
+# ABSTRACT: WHO IS CHAMP? A simple instant answer that informs people of who the champ really is.
+# See https://duck.co/ia/view/john_cena
+
+use strict;
+use DDG::Goodie;
+
+triggers start => 
+    'who is champ', 
+    'who\'s champ',
```
Comment (GuiltyDolphin) [inline lib/DDG/Goodie/JohnCena.pm]: Get rid of the escapes, use double quotes.

File: lib/DDG/Goodie/JohnCena.pm [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+# See https://duck.co/ia/view/john_cena
+
+use strict;
+use DDG::Goodie;
+
+triggers start => 
+    'who is champ', 
+    'who\'s champ', 
+    'what is his name', 
+    'what\'s his name',
```
Comment (GuiltyDolphin) [inline lib/DDG/Goodie/JohnCena.pm]: Lose these ones - I feel they are way too generic.

**Comment by moollaza** [comment]
Thanks! This LGTM :+1:

**Comment by JoshuaRLi** [comment]
Awesome! :)

**Pull request merged**